### PR TITLE
Resolve internal server error

### PR DESCRIPTION
**Pull request by AtwijukireBrendah**
# Ticket No
- https://trello.com/c/26CTbZVe/5-resolve-internal-server-error

# Summary of changes
-Added a line of code in src-data(type.tsx) and DashboardFilter 

# How should this be tested? [Screenshots, Video or Type instructions]
- After logging onto the dashboard, reload the page and the internal server error should not be appearing again.

# Notes for reviewers
- 

# Out of scope
-

### CODE REVIEW

File: src/modules/dashboard/DashboardFilter.tsx [hunk 1]
```diff
@@ -24,6 +26,8 @@ const DashboardFilter = ({ onFilter }: IProps) => {
     onFilter,
     comboFields: ['groupIdList'],
   });
+  const profile = useSelector((state: IState) => state.core.user);
+//   console.log(profile);
```
Comment (jacksono) [inline src/modules/dashboard/DashboardFilter.tsx]: Remove this console log statement.

File: src/modules/dashboard/DashboardFilter.tsx [hunk 2]
```diff
@@ -52,6 +56,7 @@ const DashboardFilter = ({ onFilter }: IProps) => {
                     remote={remoteRoutes.groupsCombo}
                     name="groupIdList"
                     label="Groups"
+					filter={{ churchName: profile.aud}}
```
Comment (jacksono) [inline src/modules/dashboard/DashboardFilter.tsx]: This seems to be over indented.

**Review by jacksono — CHANGES_REQUESTED** [review]
Some few things to work on before we can move forward with this.

**Comment by kakoma** [comment]
Combined into [this single PR](https://github.com/kanzucodefoundation/project-zoe-client/pull/119)

**Pull request closed without merging**